index.js

```diff
@@ -115,15 +115,32 @@ async function execute (inputConfig, githubConfig) {
     return resp.id;
 }
 
+function getEnv (name) {
+    const val = process.env[name] || '';
+    if (!val) {
+        throw new Error(`Environment variable expected, but not supplied: ${name}`);
+    }
+    return val;
+}
+
+function getBooleanEnv (name) {
+    const trueValue = ['true', 'True', 'TRUE'];
+    const falseValue = ['false', 'False', 'FALSE'];
+    const val = getEnv(name);
+    if (trueValue.includes(val)) return true;
+    if (falseValue.includes(val)) return false;
+    throw new TypeError(`Input could not be converted to boolean value: ${name}`);
+}
```

Comment:
Slightly more generic way of doing the same
```
function getBooleanEnv(name) {
  const val = getEnv(name).toLowerCase();
  if (val === 'true') return true;
  if (val === 'false') return false;
  throw new TypeError(`Input could not be converted to boolean value: ${name}`);
}
```

Reply:
:thinking: These two variants are not equivalent to each other, as `.toLowerCase()` would accept invalid onces (e.g. `FaLsE`), while the first one would not. And those are invalid yaml values.

Reply:
Oh, wasn't aware that yaml has these 3 variants as options, thought it doesn't care about the case. Feel free to ignore this then